src/inverseRobot.js

```diff
@@ -8,6 +8,23 @@
 
 function inverseRobot(robot) {
   // write code here
+
+  const reversed = {};
+  const check = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (check[value] !== undefined) {
```

Comment:
you can use robot object for that, please check [checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) point 2